src/Objs/Page/PageLayoutComponent.tsx

```diff
@@ -1,23 +1,6 @@
-import {
-  CurrentPage,
-  ensureUserIsLoggedIn,
-  isUserLoggedIn,
-  provideLayoutComponent,
-} from 'scrivito'
+import { CurrentPage, provideLayoutComponent } from 'scrivito'
 import { Page } from './PageObjClass'
 
-provideLayoutComponent(Page, ({ page }) => {
-  if (page.get('requireUserLogin')) {
-    ensureUserIsLoggedIn()
-
-    if (!isUserLoggedIn()) {
-      return (
-        <div className="text-center">
-          <div className="loading-placeholder" />
-        </div>
-      )
-    }
-  }
-
+provideLayoutComponent(Page, () => {
   return <CurrentPage />
```

Comment:
Given that this LayoutComponent now only renders the current page, we can remove the `PageLayoutComponent` completely.